examples/inference/benchmark_llama.py

```diff
@@ -51,6 +51,22 @@
         num_key_value_heads=40,
         max_position_embeddings=4096,
     ),
+    "llama3-8b": transformers.LlamaConfig(
+        hidden_size=4096,
+        intermediate_size=14336,
+        num_attention_heads=32,
+        num_hidden_layers=32,
+        num_key_value_heads=8,
+        max_position_embeddings=8192,
+    ),
+    "llama3-70b": transformers.LlamaConfig(
```

Comment:
Can our current framework run llama3 70B?

Reply:
Of course